backend/penndata/management/commands/get_college_house_events.py

```diff
@@ -0,0 +1,140 @@
+                self.scrape_calendar_page(f"{site}/calendar/{next_year}-{next_month:02d}")
+
+        self.stdout.write("Uploaded Rodin College House Events!")
+
+    def scrape_details(self, event_url):
+        try:
+            resp = requests.get(event_url)
+        except ConnectionError:
+            print("Error:", ConnectionError)
+            return None
+        soup = BeautifulSoup(resp.text, "html.parser")
+
+        location = (
+            soup.select_one(".field-name-field-room").text.strip()
+            if soup.select_one(".field-name-field-room")
+            else ""
+        )
+        start_time_str = (
+            soup.select_one(".date-display-start").get("content")
+            if soup.select_one(".date-display-start")
+            else ""
+        )
+        end_time_str = (
+            soup.select_one(".date-display-end").get("content")
+            if soup.select_one(".date-display-end")
+            else ""
+        )
+        start_time = (
+            datetime.datetime.strptime(start_time_str, "%Y-%m-%dT%H:%M:%S%z")
+            if start_time_str
+            else None
+        )
+        end_time = (
+            datetime.datetime.strptime(end_time_str, "%Y-%m-%dT%H:%M:%S%z")
+            if end_time_str
+            else None
+        )
+        description = (
+            soup.select_one(".field-name-body").text.strip()
+            if soup.select_one(".field-name-body")
+            else ""
+        )
+        return location, start_time, end_time, description
+
+    def scrape_calendar_page(self, calendar_url):
+        try:
+            resp = requests.get(calendar_url)
+        except ConnectionError:
+            return None
+        soup = BeautifulSoup(resp.text, "html.parser")
+
+        event_cells = soup.find_all("td", class_="single-day future")
+
+        for cell in event_cells:
+            item = cell.find("div", class_="item")
+            if not item:
+                continue
+            event_link = item.find("a", href=True)
+            if not event_link:
+                continue
+            name = event_link.text.strip()
+            url = event_link["href"]
+            index = calendar_url.find("/", calendar_url.find("://") + 3)
+            base_url = calendar_url[:index]
+            url = f"{base_url}{url}"
+
+            location, start_time, end_time, description = self.scrape_details(url)
+
+            house = calendar_url.split("/")[2].split(".")[0]
+
+            event_type = EVENT_TYPE_MAP.get(house, None)
+            Event.objects.update_or_create(
+                name=name,
+                defaults={
+                    "event_type": event_type,
+                    "image_url": "",
+                    "start": timezone.make_aware(start_time),
+                    "end": timezone.make_aware(end_time),
+                    "location": location,
+                    "website": url,
+                    "description": description,
+                    "email": "",
```

Comment:
For these and other `email` fields, I would suggest making them `None` rather than empty string just because `None` is the catch-all "there is nothing here". Since the DB fields are nullable we should be utilizing that